File: src/_score/score_module.php
<?php
/**
 * Materia
 * It's a thing
 *
 * @package	    Materia
 * @version    1.0
 * @author     UCF New Media
 * @copyright  2011 New Media
 * @link       http://kogneato.com
 */


/**
 * NEEDS DOCUMENTATION
 *
 * The widget managers for the Materia package.
 *
 * @package	    Main
 * @subpackage  scoring
 * @category    Modules
  * @author      ADD NAME HERE
 */

namespace Materia;

class Score_Modules_WordSearch extends Score_Module
{
	public function check_answer($log)
	{
		if (isset($this->questions[$log->item_id]))
		{
			$q = $this->questions[$log->item_id];
			foreach ($q->answers as $answer)
			{
				if (strtolower($log->text) == strtolower($answer['text'])) return 100;
			}
		}

		return 0;
	}

}

File: src/_score/score_module.py
import logging
from scoring.module import ScoreModule

logger = logging.getLogger("django")

class WordSearch(ScoreModule):

    def __init__(self, play=None):
        super().__init__(play)

    def check_answer(self, log):
        q = self.get_question_by_item_id(log.item_id)
        if q is not None:
            for answer in q["answers"]:
                if log.text.lower() == answer["text"].lower(): return 100
        return 0